src/Symfony/Component/DependencyInjection/ServiceLocator.php

```diff
@@ -53,13 +52,12 @@ public function get($id)
             throw new ServiceCircularReferenceException($id, array($id, $id));
```

Comment:
Do we really need to cover this? The ServiceLocator is not passed to the factories so it is very unlikely to happen.